chapters/introduction.tex

```diff
@@ -117,6 +97,10 @@ \section{Notation}\label{notation}
 end Example;
 \end{lstlisting}
 
+<<<<<<< HEAD
```

Comment:
This seems like an incomplete merge.

Reply:
Indeed.  What kind of tool allows me to commit that without a loud alarm?!

Fixed.